docs/reference/indices/index-templates.asciidoc

```diff
@@ -548,6 +550,16 @@ PUT /_index_template/template_1
 In this case, an index matching `t*` will have three primary shards. If the order of composed
 templates were reversed, the index would have two primary shards.
 
+Mapping definitions are merged recursively, which means that later mapping components can
```

Comment:
I didn't add examples here because there are many different behaviors at play and it would require a lot of space. I figured that users could call the 'simulate API' to test out individual questions they had.